README.md

```diff
@@ -1,28 +1,46 @@
 # audiorecorder
 
-## A work in progress to create a free tool for the calibration and recording of analog audio signals
+####  A work in progress to create a free tool for the calibration and recording of analog audio signals
 
-Can be installed via Homebrew: `brew tap amiaopensource/amiaos` and `brew install audiorecorder`.
+## Development information:
+For updates and history of development please see [this page](https://github.com/amiaopensource/audiorecorder/blob/master/history.md).
 
-Current Usage: [-p] passthrough mode, [-e] edit config, [-m] edit metadata for BWF insertion
+## Installation:
+Can be installed on macOS via Homebrew with the following command: 
 
-# Huge thanks to all contributors!
-## Contributors to date:
-privatezero (Andrew Weaver), retokromer (Reto Kromer), dericed (Dave Rice)
+`brew tap amiaopensource/amiaos && brew install audiorecorder`.
 
-Special thanks to Matt Boyd at the University of Washington for extensive testing assistance!
+Linuxbrew install testing is currently being conducted.
+
+
+## Usage:
+Usage: [-p] passthrough mode, [-e] edit config, [-m] edit metadata for BWF insertion
+
+#### Initial setup:
+To set up audiorecorder, first run the command `audiorecorder -e`. This will open an interface where settings such as recording bit depth and sample rate can be selected. This is also where the option to embed BEXT metadata can be selected.
+
+To set the metadata that will be embedded in the BEXT chunk (if activated in the coniguration menu) run the command `audiorecorder -m`. This will open an interface where the desired values can be entered.
 
-## Current quirks:
+#### Explanation of Interface:
+1: Peak volume meters (dB), 2: Audio spectrum - lines (from left) represent 1, 5, 10, 15, 20kHz respectively, 3: Recording time, 4: Spectrogram, 5: Audio vecorscope, 6: Graph of peak volume values (dB) from -30 dB to 0 dB. Colored area represents -5 dB to 0 dB.
```

Comment:
I would put each number on one line, because I guess it’s more readable:

1. Peak volume meters (dB)
1. Audio spectrum - lines (from left) represent 1, 5, 10, 15, 20 kHz respectively
1. Recording time
1. Spectrogram
1. Audio vecorscope.
1. Graph of peak volume values (dB) from -30 dB to 0 dB. Colored area represents -5 dB to 0 dB.

And use `20 kHz` (with a space) or cancel the space in `-30 dB`.